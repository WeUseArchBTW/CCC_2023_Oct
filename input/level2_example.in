10
3
1 2 16
3
1 8 10
3
1 5 8
2
1 2
1
9
3
1 5 16
1
5
3
1 15 16
2
1 13
5
1 3 13 69 420
