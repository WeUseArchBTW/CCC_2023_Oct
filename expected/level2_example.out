19
19
14
3
9
22
5
32
14
506
